worlds/noita/Locations.py

```diff
@@ -161,17 +219,12 @@ class Bosses(IntEnum):
     },
 }
 
-# todo: find a way to get these location counts without manually reading through, or remove the need for them
-num_static_locations = sum([len(locs) for locs in location_region_mapping.values()]) - TotalLocations.range_end  # 58
-total_locations_generated = sum([len(locs) for locs in location_region_mapping.values()]) - num_static_locations  # 500
-orb_mp_locations = 5
-orb_mw_locations = 6
-boss_mp_locations = 5
-boss_mw_locations = 7
-shop_locations = 35
 
-# length of 558 entries
 location_name_to_id: Dict[str, int] = {}
 for location_group in location_region_mapping.values():
     for locname, locinfo in location_group.items():
-        location_name_to_id.update({locname: locinfo.id})
+        if locname.endswith("Chest") or locname.endswith("Pedestal"):
```

Comment:
We should use the same mechanism here as we do in `Regions.py`, something like `if locitem.ltype in ["hc", "peds"]:`

Mainly to avoid cases later if we (or someone down the line) puts things in unique chests/pedestals. Comments indicating which `ltype` is above `location_region_mapping` would help too.

Reply:
Updated and tested